client/settings/digital-wallets/index.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardBody,
+	CardDivider,
+	CheckboxControl,
+} from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+const DigitalWallets = () => {
+	const [ isEnabled, setIsEnabled ] = useState( false );
+	const [ sectionsStatus, setSectionsStatus ] = useState( {
+		checkout: true,
+		productPage: true,
+		cart: true,
+	} );
+
+	const makeHandleSectionStatusChange = ( section ) => ( status ) => {
+		setSectionsStatus( ( oldStatuses ) => ( {
+			...oldStatuses,
+			[ section ]: status,
+		} ) );
+	};
+
+	return (
+		<Card className="digital-wallets">
+			<CardBody size="large">
+				<CheckboxControl
+					checked={ isEnabled }
+					onChange={ setIsEnabled }
+					label={ __(
+						'Enable digital wallets & express payment methods',
+						'woocommerce-payments'
+					) }
+					help={ interpolateComponents( {
+						mixedString: __(
+							// eslint-disable-next-line max-len
+							"By enabling this feature, you agree to {{stripeLink}}Stripe{{/stripeLink}}, {{appleLink}}Apple{{/appleLink}}, {{googleLink}}Google{{/googleLink}} and {{microsoftLink}}Microsoft{{/microsoftLink}}'s terms of use.",
+							'woocommerce-payments'
+						),
+						components: {
+							stripeLink: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://stripe.com/apple-pay/legal"
+								/>
+							),
+							appleLink: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://developer.apple.com/apple-pay/acceptable-use-guidelines-for-websites/"
+								/>
+							),
+							googleLink: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://androidpay.developers.google.com/terms/sellertos"
+								/>
+							),
+							microsoftLink: (
+								// eslint-disable-next-line jsx-a11y/anchor-has-content
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://www.microsoft.com/en/servicesagreement/"
+								/>
+							),
+						},
+					} ) }
+				/>
+				<h4>
+					{ __(
+						'Show digital wallets & express payment methods on:',
+						'woocommerce-payments'
+					) }
+				</h4>
+				<ul>
+					<li>
+						<CheckboxControl
+							disabled={ ! isEnabled }
+							checked={ isEnabled && sectionsStatus.checkout }
```

Comment:
Confirmed with UX - the intent is to keep the checkboxes disabled and un-checked until the feature is enabled. Once it's enabled, the merchant can check/uncheck them.

Reply:
FWIW, I agree that (at least on my screen) it's hard to determine the change of contrast between the enabled/disabled checkboxes (compared to Figma).
I think it's definitely something that can be adjusted within Gutenberg, so that all the checkboxes can benefit from the changes 👍